### DISCUSSION TITLE
Is there a way to get all the videos and gifs of the tweets?

### DISCUSSION THREAD

**Discussion by ghost** [tweepy/tweepy#2012] [Questions] [answered]
Multiple videos can now be attached to a single tweet on Twitter.
However, tweepy can only retrieve the first video.
Is there a way to get all the videos and gifs?

**Comment by Harmon758** [comment, marked as answer]
I'm not sure why you're saying Tweepy can only retrieve the first video and then proceeding to contradictorily ask for a way to retrieve all the videos, but that's incorrect anyway. Tweepy fully supports mixed media, with no changes to the library even necessary for its release.

**Comment by genki** [comment]
Twitter API seemed randomly throttling the count of medias in the mixed media tweet.
In particular, the tweet with 4 videos provides only the first one thru the API.

**Reply by Harmon758** [reply]
You'll need to provide a reproducible example (MCVE/SSCCE) for this to be actionable.

**Reply by genki** [reply]
It's just a FYI.
